docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -13,7 +13,8 @@ to start as a service.
 
   It is recommended to deploy Trino with the Helm chart on Kubernetes or
   manually with the :doc:`Docker containers <containers>` or the :doc:`tar
-  archive <deployment>`. While the RPM is available for use, it is not tested.
+  archive <deployment>`. While the RPM is available for use, it is not
+  supported.
```

Comment:
avoid "supported". It has specific meaning to different people. I'd probably suggest saying something like "using the RPM is discouraged in favour of the tarball or Docker containers".

Reply:
I like that.